src/DssSpell.t.sol

```diff
@@ -1054,4 +1246,30 @@ contract DssSpellTest is DSTest, DSMath {
         address UNIUSD_MED  = 0x52f761908cC27B4D77AD7A329463cf08baf62153;
         assertEq(MedianAbstract(UNIUSD_MED).bud(SET_UNI), 1);
     }
+
+    function testOsmWhitelist() public {
```

Comment:
This test can be moved to `testOsms`

Add an INSTADAPP constant and use the `addr.addr("X_PIP")`, then we can leave it in there to track authorizations.